css/style.css

```diff
@@ -1,8 +1,4 @@
-/*
-primary (blue) - #0099ff
-secndary (yellow) - #ffff33
-tertiary ( dark blue) - #0039e6 
-*/
+/* need to add more colours later*/
```

Comment:
Please capitalise "N" in need and also add a full stop at the end of sentence and also add space after full stop.

Reply:
Done.